preact/main/playlists/songsList.json

```diff
@@ -0,0 +1,114 @@
+[
+  {
+    "title": "シンクロニティ by おめシス × BACK-ON",
+    "videoID": "jis7E_mbwPw"
+  },
+  {
+    "title": "【ガンダムAGE】AURORA (Short) / 藍井エイル by おめがレイ",
+    "videoID": "8q-Ny2a0GPM"
+  },
+  {
+    "title": "【ボカロ】非公開日誌 / GUMI by おめがリオ",
+    "videoID": "FkyotXkl-CI"
+  },
+  {
+    "title": "からくりピエロ / 初音ミク by おめがリオ",
+    "videoID": "owl2ZtgDRHI"
+  },
+  {
+    "title": "【天元突破グレンラガン】空色デイズ (Short) / 中川翔子 by おめがレイ",
+    "videoID": "-YLyVDOJnFc"
+  },
+  {
+    "title": "【DTB】ツキアカリのミチシルベ / ステレオポニー by おめがレイ",
+    "videoID": "SV_ag_cq9Jg"
+  },
+  {
+    "title": "青いベンチ / サスケ by おめがシスターズ",
+    "videoID": "3JAeD-9FE8I"
+  },
+  {
+    "title": "【GBF】ニブンノイチ / BACK-ON by おめがシスターズ",
+    "videoID": "KQPr8Qde22w"
+  },
+  {
+    "title": "【アルドノア・ゼロ】heavenly blue / Kalafina by おめがレイ",
+    "videoID": "IfYODh82hTo"
+  },
+  {
+    "title": "AM11:00 / HY by おめがシスターズ【#16Ω】",
+    "videoID": "abDwOIMcBEw"
+  },
+  {
+    "title": "【時をかける少女】ガーネット / 奥華子 by おめがシスターズ",
+    "videoID": "gXhqIT7XnsI"
+  },
+  {
+    "title": "夏色 / ゆず by おめがシスターズ",
+    "videoID": "VBpXOOPK6-E"
+  },
+  {
+    "title": "【かっとビング】マスターピース / mihimaru GT by おめがシスターズ",
+    "videoID": "_AcOHoF68vo"
+  },
+  {
+    "title": "【ﾗｯﾀｯﾀｧ!】独りんぼエンヴィー / 初音ミク byおめがシスターズ",
+    "videoID": "s9IvNnu0kqc"
+  },
+  {
+    "title": "【ﾒｰﾃﾞｰ】ゴーストルール / 初音ミク byおめがシスターズ",
+    "videoID": "FVfim9jNQVg"
+  },
+  {
+    "title": "【まどマギ】コネクト / ClariS by おめがシスターズ",
+    "videoID": "sYNL6WlW-e8"
+  },
+  {
+    "title": "【血界戦線】シュガーソングとビターステップ / UNISON SQUARE GARDEN by おめがシスターズ",
+    "videoID": "pSBk3QVY3cA"
+  },
+  {
+    "title": "【あおガル】恋のカトレア通り(cover) by きれいなおめシス",
+    "videoID": "b0nF7oUbsfg"
+  },
+  {
+    "title": "ホウキ雲 / RYTHEM by おめがシスターズ",
+    "videoID": "9Ppv1wvwHXI"
+  },
+  {
+    "title": "透明声彩 / YuNi by おめがシスターズ",
+    "videoID": "58al_aq_E2M"
+  },
+  {
+    "title": "【ハガレン】メリッサ / ポルノグラフィティ by おめがシスターズ【#14Ω】",
+    "videoID": "XAt2lODxz_E"
+  },
+  {
+    "title": "白日 / King Gnu by おめがシスターズ",
+    "videoID": "58TGnNnXT6E"
+  },
+  {
+    "title": "点描の唄 / Mrs. GREEN APPLE (feat.井上苑子) covered by おめがシスターズ",
+    "videoID": "dmloYn5xaqs"
+  },
+  {
+    "title": "砂の惑星 / covered by おめがシスターズ",
+    "videoID": "xCcWSD4SKJo"
+  },
+  {
+    "title": "夜に駆ける ／YOASOBI  covered by おめがシスターズ",
+    "videoID": "vNYev-RNNDs"
+  },
+  {
+    "title": "香水 ／瑛人 covered by おめがシスターズ",
+    "videoID": "OQggCK-lfOk"
+  },
+  {
+    "title": "うっせぇわ / Ado by おめがシスターズ",
+    "videoID": "fdpWxx9sqnc"
+  },
+  {
+    "title": "スピッツ ／チェリー covered by おめがシスターズ with MSE",
+    "videoID": "K4d4xE4OILM"
+  }
+]
```

Comment:
extracted json into pure json file from ts file so that gen_songs_list.ts can write result easily.

If this is not desirable change, will revert here.